src/util.cpp

```diff
@@ -5971,28 +5971,37 @@ static QCString replaceAliasArguments(StringUnorderedSet &aliasesProcessed,
   int markerEnd=0;
   for (i=0;i<l;i++)
   {
```

Comment:
The logic in the loop has become too complex in my opinion. I'm pretty sure it can be simplified. Can you list the cases you now encountered and cover?

Reply:
I think the logic has decreased, it would be possible to remove the `continue` and the second `if` can in that case be replaced with `else if`.

The major test cases I encountered are shown in the example as attached to the pull request here on github.